src/twisted/newsfragments/9359.feature

```diff
@@ -0,0 +1 @@
+Provide web.server.request to have its hyperlink.URL, as URLPath is superceded by hyperlink.URL.
```

Comment:
Minor wording here: it's a bit more natural to say something like "twisted.web.server.request now provides access to its hyperlink.URL."